hubblestack/daemon.py

```diff
@@ -178,11 +178,27 @@ def main():
 
         try:
             log.debug('Executing schedule')
-            schedule()
+            sf_count = schedule()
+            if sf_count > 0:
+                log.debug('Executed %d schedule item(s)', sf_count)
+                workaround_salt_log_handler_queues()
         except Exception as e:
             log.exception('Error executing schedule')
         time.sleep(__opts__.get('scheduler_sleep_frequency', 0.5))
 
+def workaround_salt_log_handler_queues():
```

Comment:
Would you mind moving this function to [log.py](https://github.com/hubblestack/hubble/blob/develop/hubblestack/log.py)? I want to keep every bit of complexity out of daemon.py where it makes sense.